Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -489,6 +490,23 @@ YUI.add('ez-platformuiapp', function (Y) {
         },
 
         /**
+         * Middleware to warn about the depreciation of a route. It just logs a
+         * message to console and call the `next` callback.
+         *
+         * @method logDeprecatedRoute
+         * @param {Object} req
+         * @param {Object} res
+         * @param {Function} next
+         */
+        logDeprecatedRoute: function (req, res, next) {
+            var route = req.route;
+
+            console.log('[DEPRECATED] The route "' + route.name + '" (path "' + route.path + '") is deprecated');
```

Comment:
well, the console object lacks a `deprecated` log level. To me `warn` is more for potential issues close to error (a deprecation is not really in that category). Maybe we could use `notice` but we should probably have a component responsible for logging the deprecations first because such console.log calls are spread all over the place.